Plugins/TGeo/src/TGeoLayerBuilder.cpp

```diff
@@ -152,9 +152,15 @@ void Acts::TGeoLayerBuilder::buildLayers(const GeometryContext& gctx,
 
     // Step down from the top volume each time to collect the logical tree
     TGeoVolume* tvolume = gGeoManager->GetTopVolume();
+    TGeoVolume* dvolume =
+        gGeoManager->FindVolumeFast(layerCfg.volumeName.c_str());
+    if (dvolume != nullptr) {
+      tvolume = dvolume;
+      ACTS_DEBUG("- setting search volume to " << dvolume->GetName());
+    }
```

Comment:
```suggestion
    TGeoVolume* tvolume =  gGeoManager->FindVolumeFast(layerCfg.volumeName.c_str());
    if(tVolume == nullptr) {
      tvolume = gGeoManager->GetTopVolume();
    } else {
      ACTS_DEBUG("- setting search volume to " << dvolume->GetName());
    }
```
If I made no mistake then this would remove `dvolume`.

Reply:
Correct, I will change accordingly.

Reply:
Although you'd have to change `dvolume` to `tvolume` in l.158 to make it compile.